core/commands/store/contracts/contracts.go

```diff
@@ -193,25 +200,37 @@ This command get contracts list based on role from the local node data store.`,
 		if !ok {
 			return fmt.Errorf("invalid filter option: %s", filterOpt)
 		}
-		// TODO: Use size
-		// TODO: sort
-		//sizeOpt, _ := req.Options[contractsListSizeOptionName].(int)
-		contracts, err := ListContracts(n.Repo.Datastore(), req.Arguments[0])
+		size, ok := req.Options[contractsListSizeOptionName].(int)
+		if !ok {
+			return fmt.Errorf("invalid size option: %s", size)
```

Comment:
no need to check `ok`, size always has a default, if someone passes an invalid size, it will throw error on argument parsing, not here.